rmw_connext_cpp/src/rmw_get_network_flow_endpoints.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Ericsson AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rmw/error_handling.h"
+#include "rmw/get_network_flow_endpoint.h"
```

Comment:
```suggestion
#include "rmw/get_network_flow_endpoints.h"
```

The name of the file looks wrong 
https://github.com/Ericsson/ros2-rmw/blob/c1c25072c199c3177121768a612243386b84b054/rmw/include/rmw/get_network_flow_endpoints.h#L1 (?)

Reply:
@ivanpauno Thanks for noticing the mistake. I understood that this repository is going to be deprecated so did not include it in my integration recent tests with Rolling Ridley.  Since this PR got merged, do I go ahead and create a new PR to fix the error?

Reply:
> Since this PR got merged, do I go ahead and create a new PR to fix the error?

Yes, thanks!
We don't need to continue adding new features here, but if we do we shouldn't break it.
(I know that there are people still using this package).

Reply:
I submitted a [PR](https://github.com/ros2/rmw_connext/pull/493) with corrections. Sorry about overlooking this. 

Please ensure that it passes compile-time checks and does not break functionality. I am unable to test fully due to lack of a licensed RTI Connext system at my side.